api/checks/api.go

```diff
@@ -49,3 +53,7 @@ func (s suitesAPIImpl) PendingCheckRun(sha string, product shared.ProductSpec) (
 func (s suitesAPIImpl) CompleteCheckRun(sha string, product shared.ProductSpec) (bool, error) {
 	return completeCheckRun(s.ctx, sha, product)
 }
+
+func (s suitesAPIImpl) ScheduleResultsProcessing(sha string, product shared.ProductSpec) error {
+	return scheduleResultsProcessing(s.ctx, sha, product)
```

Comment:
I don't see `scheduleResultsProcessing` itself being used anywhere else. Can we just embed the implementation here in this exported method?